apps/testingaccessibility/src/pages/team/index.tsx

```diff
@@ -95,6 +99,14 @@ const TeamPage: React.FC<React.PropsWithChildren<TeamPageProps>> = ({
             setPersonalPurchase={setPersonalPurchase}
           />
         </Card>
+        <Card
+          title={{content: 'Get more seats', as: 'h2'}}
+          icon={
+            <TicketIcon className="w-5 text-green-500" aria-hidden="true" />
+          }
+        >
+          <BuyMoreSeats productId={purchase.product.id} userId={userId} />
+        </Card>
```

Comment:
From the team page, render the `BuyMoreSeats` component with the `productId` of the existing purchase and this user's `id`.

I think we are fine to only render this here and not also on the welcome page which looks very similar. That feels unnecessarily pushy to have it there too when they have _just_ made their initial purchase.

Reply:
ya, we don't want to sell them, but we should make sure to mention how they can add more seats via email! 

a future iteration could have this available for individuals as well so they can "upgrade" to a bulk account. we've definitely had folks do that!